sdks/python/apache_beam/io/range_trackers.py

```diff
@@ -77,6 +79,10 @@ def stop_position(self):
   def last_record_start(self):
     return self._last_record_start
 
+  @property
+  def last_attempted_record_start(self):
```

Comment:
Please add documentation about this function here.

Reply:
I added a comment about last_attempted_record_start here: https://github.com/apache/beam/pull/9058/files#diff-6a5b8165b53882ae05785d5a4ccfe9cbR64. Is it enough to illustrate what a `last_attempted_record_start` is? I think the function itself is simple enough and self-illustrative.

Reply:
A pydoc for the property (on the getter) is much better :)

Reply:
For example: "Records a position that tried be claimed by the user of this range tracker as long as it's a valid position. This will be updated for every 'try_claim()' call, irrespective of whether the call is successful or not. This will not be updated for 'set_current_position()' calls.

Reply:
https://github.com/apache/beam/pull/9058/commits/6ad7c4f277a745c3985c561e3a1eb37e814e3e5c